### PR TITLE
Fix DHCP on restart [full ci]

### PR DESCRIPTION
**Pull request by hmahmood**

### CODE REVIEW

**Comment by andrewtchin** [comment]
lgtm

File: lib/config/executor/network_interface.go [hunk 1]
```diff
@@ -60,17 +60,22 @@ type ContainerNetwork struct {
...
+	Nameservers []net.IP `vic:"0.1" scope:"read-only" key:"dns"`
 
 	// The IP ranges for this network
 	Pools []ip.Range `vic:"0.1" scope:"read-only" key:"pools"`
 
 	// set of network wide links and aliases for this container on this network
 	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
+
+	Assigned struct {
+		Gateway     net.IPNet `vic:"0.1" scope:"read-write" key:"gateway"`
```
Comment (hickeng) [inline lib/config/executor/network_interface.go]: for what it's worth, you don't need to have the annotations on the elements - they'll inherit from the struct

**Comment by vburenin** [comment]
LGTM

**Comment by hickeng** [comment]
lgtm

**Comment by hmahmood** [comment]
:recycle:

**Pull request merged**